Specs/Number/English/PercentModelPercentMode.json

```diff
@@ -0,0 +1,141 @@
+[ 
+{
+  "Input": "100%",
+  "NotSupported": "javascript,python",
+  "Results": [
+    {
+      "Text": "100%",
+      "TypeName": "percentage",
+      "Resolution": {
+        "value": "100%"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "twenty percent",
+  "NotSupported": "javascript,python",
+  "Results": [
+    {
+      "Text": "twenty percent",
+      "TypeName": "percentage",
+      "Resolution": {
+        "value": "20%"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "one hundred percents",
+  "NotSupported": "javascript,python",
+  "Results": [
+    {
+      "Text": "one hundred percents",
+      "TypeName": "percentage",
+      "Resolution": {
+        "value": "100%"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "percents of twenty",
```

Comment:
I don't think this is valid English.

Reply:
I copy the tests from origin number model tests.